src/pages/country/[countryCode]/index.tsx

```diff
@@ -0,0 +1,139 @@
+import { useContext } from "react";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import { useQuery } from "@tanstack/react-query";
+import { BorderCountryBox } from "@/components";
+import { searchCountryByCode } from "@/server/actions";
+import { ThemeContext } from "@/context/context";
+import { CircularProgress } from "@mui/joy";
+import { formatCapitalsDisplay } from "@/components/CountryCard";
+
+function DetailPage() {
+  const router = useRouter();
+  const countryCode = router.query.countryCode;
+  const { data, isLoading } = useQuery({
+    queryKey: ["countries", countryCode],
+    queryFn: () => searchCountryByCode(countryCode?.toString()!),
+  });
+  let currentTheme = useContext(ThemeContext);
+  return (
+    <div
+      className={`app__detail ${
+        currentTheme == "Dark Mode"
+          ? "section_dark dark"
+          : "section_light light"
+      }`}
+    >
+      <Link href="/">
```

Comment:
@jakemelvin, NextJS has a built-in router hook that you can use. You can create a `const router = useRouter()` and then use a button with an onClick function that calls `router.back()` to navigate backwards.